src/audio/volume/Kconfig

```diff
@@ -76,4 +76,25 @@ config COMP_GAIN
 	  This option enables gain to change volume. It works
 	  as peak volume without updating peak vol to host
 
+config VOLUME_HIFI4
+    bool
+    default y if ACE
```

Comment:
Hmm, this won't work unless the toolchain has hifi support. E.g. Zephyr SDK supports ACE but can't handle hifi code.